### DISCUSSION TITLE
Is there an online example of the multimodal (bike, bikeshare, public transit) function?

### DISCUSSION THREAD

**Discussion by sbutler-gh** [valhalla/valhalla#3608] [Q&A] [answered]
When trying the https://valhalla.openstreetmap.de/ webapp, I couldn't find an example of the multimodal functionality.  Is there any demo or online example where the multimodal – specifically including bikes, bike share, public transit, buses — is available and can be used?

**Comment by kevinkreiser** [comment, marked as answer]
I doest have any multimodal data loaded at the moment. We are working on fixing that via #3539

**Reply by sbutler-gh** [reply]
Got it, thanks so much for input @kevinkreiser !
